src/core/components/Footer/footer.less

```diff
@@ -117,3 +117,7 @@
 #footer-map div a img {
   display: none;
 }
+
+.social > a:last-child {
+  margin-right: 50px;
+}
```

Comment:
Sorry about not getting this in the first review, but if you have a look at the nested selector at line 21, there is already code for selecting almost this element. Nesting the ``:last-child` in there would provide for a bit of a cleaner solution?

Reply:
Agree, fixed it now